app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -142,6 +146,9 @@ open class FenixApplication : LocaleAwareApplication() {
         // if ((System.currentTimeMillis() - settings().lastPlacesStorageMaintenance) > ONE_DAY_MILLIS) {
         //    runStorageMaintenance()
         // }
+
+        onRegisterLifecycleCallback = OnRegisterActLifeCycleCallback(components.performance)
```

Comment:
nit: we don't use this reference for anything else: why do we need to hold on to it?